fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/derby/DerbyResourceDAO.java

```diff
@@ -203,9 +203,9 @@ public Resource  insert(Resource resource, List<ExtractedParameterValue> paramet
      * @return the resource_id for the entry we created
      * @throws Exception
      */
-    public long storeResource(String tablePrefix, List<ExtractedParameterValue> parameters, 
+    public long storeResource(String tablePrefix, List<ExtractedParameterValue> parameters,
```

Comment:
There's also PostgresResourceNoProcDAO which I try to keep in-sync - it is handy sometimes to use instead of the stored proc when investigating issues. It replicates the logic of the stored procedure using discrete statements.